src/components/TodosList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+function TodosList({
+  todosProps, handleChange, delTodo, setUpdate,
```

Comment:
[ ] optional: It is fine to get props like this but when you use liners, it will show an error 😥 . So it would be better to use the props keyword here.